### PR TITLE
[hi] localize content/en/docs/reference/glossary/limitrange.md

### PR DESCRIPTION
**Pull request by chiukapoor**
Signed-off-by: Chirayu Kapoor <dev.csociety@gmail.com>

Localized the content of `content/en/docs/reference/glossary/limitrange.md` into Hindi language in the file `content/hi/docs/reference/glossary/limitrange.md`

Fixes https://github.com/kubernetes/website/issues/37856 and https://github.com/kubernetes/website/issues/35600

### CODE REVIEW

**Comment by chiukapoor** [comment]
Hello @divya-mohan0209 and @anubha-v-ardhan, Hope you are doing well!

May you please review the changes and let me know your feedback?

File: content/hi/docs/reference/glossary/limitrange.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+related:
+ - pod
+ - container
+
+---
+ नामस्थान (namespace) में प्रति {{< glossary_tooltip text="कंटेनर" term_id="container" >}} या {{< glossary_tooltip text="पॉड" term_id="pod" >}} में संसाधन खपत को सीमित करने के लिए प्रतिबंध प्रदान करता है।
+
+<!--more--> 
+लिमिटरेंज (LimitRange) टाइप (type) द्वारा बनाई जा सकने वाली वस्तुओं की मात्रा को सीमित रहता है,
+साथ ही एक नामस्थान (namespace) में अलग-अलग {{< glossary_tooltip text="कंटेनर" term_id="container" >}} या {{< glossary_tooltip text="पॉड" term_id="pod" >}} द्वारा अनुरोधित/उपभोग किए जा सकने वाले कंप्यूट संसाधनों की मात्रा को भी।
```
Comment (bishal7679) [inline content/hi/docs/reference/glossary/limitrange.md:23]:
```suggestion
लिमिटरेंज, टाइप (type) द्वारा बनाई जा सकने वाले ऑब्जेक्ट्स और साथ ही नेमस्पेस में अलग-अलग {{< glossary_tooltip text="कंटेनर" term_id="container" >}} या {{< glossary_tooltip text="पॉड" term_id="pod" >}} द्वारा अनुरोध/उपभोग किए जा सकने वाले कंप्यूट संसाधनों की मात्रा को सीमित करता है।
```

File: content/hi/docs/reference/glossary/limitrange.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
+---
+title: लिमिटरेंज (LimitRange)
+id: limitrange
+date: 2019-04-15
+full_link:  /docs/concepts/policy/limit-range/
+short_description: >
+  नामस्थान (namespace) में प्रति कंटेनर या पॉड में संसाधन खपत को सीमित करने के लिए प्रतिबंध प्रदान करता है।
```
[the remark this answers is not in the document]
Comment (chiukapoor) [reply content/hi/docs/reference/glossary/limitrange.md]: Thank you for the suggestions, inculcate the changes.

File: content/hi/docs/reference/glossary/limitrange.md [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+tags:
+- core-object
+- fundamental
+- architecture
+related:
+ - pod
+ - container
+
+---
+ नामस्थान (namespace) में प्रति {{< glossary_tooltip text="कंटेनर" term_id="container" >}} या {{< glossary_tooltip text="पॉड" term_id="pod" >}} में संसाधन खपत को सीमित करने के लिए प्रतिबंध प्रदान करता है।
```
[the remark this answers is not in the document]
Comment (chiukapoor) [reply content/hi/docs/reference/glossary/limitrange.md]: Thank you for the suggestions, inculcate the changes.

File: content/hi/docs/reference/glossary/limitrange.md (hunk 1 above)
Comment (chiukapoor) [reply content/hi/docs/reference/glossary/limitrange.md]: Thank you for the suggestions, inculcate the changes.

**Comment by chiukapoor** [comment]
Hello @divya-mohan0209 and @anubha-v-ardhan, PTAL.

**Review by divya-mohan0209 — COMMENTED** [review]
/lgtm

**Review by kundan2707 — COMMENTED** [review]
/lgtm

**Review by Garima-Negi — APPROVED** [review]
/LGTM

**Comment by divya-mohan0209** [comment]
Taking the approval from the comment [here](https://github.com/kubernetes/website/pull/37872#issuecomment-1323280580), signing off for merge.

/approve

**Pull request merged**